web/src/stream_data.js

```diff
@@ -846,3 +846,25 @@ export function initialize(params) {
 export function remove_default_stream(stream_id) {
     default_stream_ids.delete(stream_id);
 }
+
+export function options_for_dropdown_widget() {
+    return (
+        subscribed_subs()
+            .map((stream) => ({
+                name: stream.name,
+                unique_id: stream.stream_id,
+                stream,
+            }))
+            // eslint-disable-next-line consistent-return, array-callback-return
+            .sort((a, b) => {
+                if (a.name.toLowerCase() < b.name.toLowerCase()) {
```

Comment:
I understand that the changes I'm mentioning are beyond the scope of this PR, but I wanted to share them anyway.

We can use the [localeCompare()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare) method here to simplify the code. The updated code would look like this:

`.sort((a, b) => a.name.toLowerCase().localeCompare(b.name.toLowerCase()))`

This code achieves the same result as the original code but in a more concise and readable manner.

Reply:
Added as a separate commit. Thanks!